packages/pangea-sdk/pangea/utils.py

```diff
@@ -1,5 +1,8 @@
 import base64
+import collections as c
```

Comment:
nitpick: rather than rename collections to c, why not import OrderedDict etc from collections

Reply:
Done!